d2/src/main/java/com/linkedin/d2/xds/XdsToD2PropertiesAdaptor.java

```diff
@@ -221,11 +221,18 @@ public void onChanged(XdsClient.NodeUpdate update)
           }
           catch (PropertySerializationException e)
           {
+            _serviceEventBus.publishInitialize(serviceName, null); // still notify event bus to avoid timeout in case some subscribers are waiting for the data
```

Comment:
```suggestion
            // still notify event bus to avoid timeout in case some subscribers are waiting for the data
            _serviceEventBus.publishInitialize(serviceName, null);
```

Reply:
Done